extension/syntaxes/templateSettingsSyntax.json

```diff
@@ -0,0 +1,207 @@
+                  "title": "Match",
+                  "description": "Specifies the text that will be replaced in the filename",
+                  "type": "string",
+                  "default": "",
+                  "minLength": 1,
+                  "examples": [],
+                  "pattern": "^.+$"
+                },
+                "removeSpaces": {
+                  "$id": "#root/mappings/items/renameFile/items/removeSpaces",
+                  "title": "Remove Spaces",
+                  "description": "Specifies if any whitespace in the text that is used for replacing will get any whitespace removed.",
+                  "type": "boolean",
+                  "examples": [],
+                  "default": false
+                }
+              }
+            }
+          },
+          "searchAndReplace": {
+            "$id": "#root/mappings/items/searchAndReplace",
+            "title": "Search and replace",
+            "description": "Specifies any settings for Search and replace text in files",
+            "type": "array",
+            "default": [],
+            "items": {
+              "$id": "#root/mappings/items/searchAndReplace/items",
+              "title": "Items",
+              "type": "object",
+              "required": [
+                "path",
+                "match"
+              ],
+              "properties": {
+                "path": {
+                  "$id": "#root/mappings/items/searchAndReplace/items/path",
+                  "description": "Specifies the file path filter (glob pattern) that is used to select which files will be used for search & replace",
+                  "title": "Path",
+                  "type": "string",
+                  "default": "**/*",
+                  "examples": [
+                    "**/*"
+                  ],
+                  "pattern": "^.*$"
+                },
+                "match": {
+                  "$id": "#root/mappings/items/searchAndReplace/items/match",
+                  "description": "Specifies the text that will be replaced",
+                  "title": "Match",
+                  "type": "string",
+                  "minLength": 1,
+                  "default": "",
+                  "examples": [],
+                  "pattern": "^.*$"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "createXlfLanguages": {
+      "$id": "#root/createXlfLanguages",
+      "title": "Create XLF Languages",
+      "description": "Specifies the language codes for the languages that XLF files should be created for. (Currently supported languages are found at https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/compliance/apptest-countries-and-translations)",
+      "type": "array",
+      "default": [],
+      "items": {
+        "$id": "#root/createXlfLanguages/items",
+        "title": "Items",
+        "type": "string",
+        "default": "",
+        "examples": [
+          "bg-BG",
```

Comment:
Would it be possible to extract this to a separate definition to be reused by other schemas?

Reply:
I don't know how to do that.. 🙂

Save until we need that?

Reply:
Ye, we'll look into that  at some point. I now it's possible within one schema but I wouldn't know how to share it between different schemas.